packages/graphql/src/contracts.js

```diff
@@ -168,7 +168,7 @@ export function setNetwork(net, customConfig) {
       breakdownEvery,
       maxConcurrent,
       qps,
-      ethGasStation: net === 'mainnet' ? true : false
+      ethGasStation: ['mainnet', 'rinkeby'].indexOf(net) > -1 ? true : false
```

Comment:
nit: a maybe slightly simpler syntax would be:
['mainnet', 'rinkeby'].includes(net)

Reply:
simplified!